### DISCUSSION TITLE
Possible to use swift-parsing without the case paths macro?

### DISCUSSION THREAD

**Discussion by mikewoodworth** [pointfreeco/swift-parsing#362] [General]
Hey guys, I'm dying trying to use this package on windows... the addition on macros is adding over 5 minutes to my compiles. Is it possible to use the package somehow without pulling the case path dependancy?

Thanks,
mike

**Comment by stephencelis** [comment]
@mikewoodworth You can pin the Case Paths project to a pre-macro version, which is 1.0.0.

I'd also recommend filing feedback and GitHub issues with Apple. The Swift core team has heavily promoted the feature, so if you find it so unusable that you avoid packages that contain macros, it's something that they should know about and prioritize fixing.
